scripts/prohibit-inner-classes

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+if git grep -En '^ +[^/*"]*\bclass ' -- '*.java' | grep -v '\bstatic '; then
```

Comment:
It's likely doable with Checkstyle `Regexp`, but would require negative lookbehinds or some other brittle stuff